### ISSUE TITLE
Can't "bmcs os object put" from stdin

### ISSUE THREAD

**Issue by bgilbert** [oracle/bmcs-cli#7]
`bmcs os object put -?` says:

```
  --file FILENAME                 The file to load as the content of the
                                  object, or '-' to read from STDIN.
```

But on Fedora 26:

```
$ bmcs os object put -ns $ns -bn $bucket --name test --file - <<<data
FileNotFoundError: [Errno 2] No such file or directory: '<stdin>'
```

**Comment by mross22** [comment]
Thanks for reporting, yes this is a regression in one of our recent releases.  We are working on a fix now and should have an update soon.

**Comment by jodoglevy** [comment]
Fixed in v2.4.6
